packages/scaffold/src/partials/w3m-email-login-widget/index.ts

```diff
@@ -71,47 +75,41 @@ export class W3mEmailLoginWidget extends LitElement {
               </wui-icon-link>
             `
           : null}
-        ${this.loading && multipleConnectors
+        ${this.loading
           ? html`<wui-loading-spinner size="md" color="accent-100"></wui-loading-spinner>`
           : null}
 
         <input type="submit" hidden />
       </form>
 
-      ${multipleConnectors
-        ? html`<wui-separator text="or"></wui-separator>`
-        : html`<wui-button
-            size="md"
-            variant="fill"
-            fullWidth
-            @click=${this.onSubmitEmail.bind(this)}
-            .disabled=${!showSubmit}
-            .loading=${this.loading}
-          >
-            Continue
-          </wui-button>`}
+      ${multipleConnectors ? html`<wui-separator text="or"></wui-separator>` : null}
     `
   }
 
   // -- Private ------------------------------------------- //
   private onEmailInputChange(event: CustomEvent<string>) {
     this.email = event.detail
+    this.error = ''
   }
 
   private async onSubmitEmail(event: Event) {
     try {
       if (this.loading) {
         return
       }
+      const { success: isEmailValid } = z.string().email().safeParse(this.email)
```

Comment:
Is this validation necessary here? When you submit invalid email it should be already validated upstream through schema of `wallet` package and throw an error that's shown to user.

Reply:
this was to prevent the api throwing an error + showing the error toast since this is a local validation and we'd need to show the input's error message. I guess I can remove this and somehow parse actual error but seemed a bit hackier than this

Reply:
effectively checked the validation type, not the cleanest but it works without relying on zod. 

might be worth reviewing what other kinds of errors could happen that should show the input error instead of the snackbar error. from Diane's notes, it's anything related to input validation but can't think of anything else other than mail formatting right now.